src/__experimental__/components/fieldset/fieldset.component.js

```diff
@@ -64,7 +78,9 @@ Fieldset.propTypes = {
   /** Prop to indicate additional information  */
   hasInfo: PropTypes.bool,
   /** A message that the ValidationIcon component will display */
-  tooltipMessage: PropTypes.string
+  tooltipMessage: PropTypes.string,
+  /** When true, legend is placed in line with the children */
+  legendInline: PropTypes.bool
 };
 
```

Comment:
```suggestion

Fieldset.defaultProps = {
  legendInline: false
}
```

Reply:
There's no mention in the PR why the `Fieldset` needs to change, can you add more detail to the PR' please?